__tests__/app.test.js

```diff
@@ -267,3 +267,54 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: request body is accepted and responds with the posted comment object", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "you are not good at writing news articles",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toMatchObject(
+          expect.objectContaining({
+            article_id: 1,
+            author: "icellusedkars",
+            body: "you are not good at writing news articles",
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
```

Comment:
I would change this to be 0, as 0 is the default value it should always be 0 when making a new comment. Also because we are working with a test database we can know exactly how many comments there are at the time that we post this so we can even be more precise with the comment_id we expect because it is a SERIAL PRIMARY KEY. (We can just count the amount in the test data comments file)